### DISCUSSION TITLE
from pascal to julia

### DISCUSSION THREAD

**Discussion by fabbius69** [JuliaLang/julia#47023] [General]
is it possible to have similar code in julia?
thanks

"type
lo=array[1..5,1..10]of smallint;
nom=string[25];
lot=record
num:lo;
ind:smallint;
anno:smallint;
mese:string[15];
end;
fi=file of lot;
var
f2,f3:fi;"

**Comment by oscardssmith** [comment]
Yes. In the future https://discourse.julialang.org, slack or zulip are better places for questions.
